apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -338,6 +344,29 @@ static AppConfiguration readConfigFromEnvironmentVariables() {
               "Missing value for configuration environment variable '%s'.",
               ENV_VAR_KEY_DATABASE_PASSWORD));
 
+    String databaseMaxPoolSizeText = System.getenv(ENV_VAR_KEY_DATABASE_MAX_POOL_SIZE);
+    Optional<Integer> databaseMaxPoolSizeOption;
+    if (databaseMaxPoolSizeText == null || databaseMaxPoolSizeText.isEmpty()) {
+      databaseMaxPoolSizeOption = Optional.empty();
+    } else {
+      try {
+        databaseMaxPoolSizeOption = Optional.of(Integer.parseInt(databaseMaxPoolSizeText));
+      } catch (NumberFormatException e) {
+        databaseMaxPoolSizeOption = Optional.of(-1);
+      }
+      if (databaseMaxPoolSizeOption.get() < 1)
+        throw new AppConfigurationException(
+            String.format(
+                "Invalid value for configuration environment variable '%s': '%s'",
+                ENV_VAR_KEY_DATABASE_MAX_POOL_SIZE, databaseMaxPoolSizeText));
+    }
+
```

Comment:
This PR probably isn't the place to address this but it shouldn't take this much code to add one more environment variable.  Something like the `getIntOrDefault()` method (but throwing if the variable is missing/invalid rather than having a default) would shrink this and similar cases down to one LOC.

Reply:
I can't believe I let myself get talked into this, but your wish is my command. Done.